bears/r/FormatRBear.py

```diff
@@ -65,24 +65,21 @@ def create_arguments(filename, file, config_file,
                 1
             ```
         """
-        rcode = ('library(formatR);'
-                 'formatR::tidy_source('
-                 'source="' + escape(filename, '"\\') + '",'
-                 'comment={r_keep_comments},'
-                 'blank={r_keep_blank_lines},'
-                 'arrow={r_use_arrows},'
-                 'brace.newline={r_braces_on_next_line},'
-                 'indent='
-                 '{tab_width}'.format(tab_width=tab_width,
-                                      r_keep_comments=_map_to_r_bool(
-                                          r_keep_comments),
-                                      r_keep_blank_lines=_map_to_r_bool(
-                                          r_keep_blank_lines),
-                                      r_use_arrows=_map_to_r_bool(
-                                          r_use_arrows),
-                                      r_braces_on_next_line=_map_to_r_bool(
-                                          r_braces_on_next_line)))
-        # Disable r_max_expression_length if it is equal to 0
-        rcode += ('' if r_max_expression_length == 0
-                  else ',width.cutoff=' + str(r_max_expression_length)) + ')'
+        options = {'source="' + escape(filename, '"\\') + '"': True,
+                   'blank=' + _map_to_r_bool(r_keep_blank_lines):
+                       r_keep_blank_lines is not None,
```

Comment:
Done :)

2016-06-16 21:51 GMT+01:00 Lasse Schuirmann <notifications@github.com>:

> In bears/r/FormatRBear.py
> <https://github.com/coala-analyzer/coala-bears/pull/525#discussion_r67421605>
> :
>
> > -                 'indent='
> > -                 '{tab_width}'.format(tab_width=tab_width,
> > -                                      r_keep_comments=_map_to_r_bool(
> > -                                          r_keep_comments),
> > -                                      r_keep_blank_lines=_map_to_r_bool(
> > -                                          r_keep_blank_lines),
> > -                                      r_use_arrows=_map_to_r_bool(
> > -                                          r_use_arrows),
> > -                                      r_braces_on_next_line=_map_to_r_bool(
> > -                                          r_braces_on_next_line)))
> > -        # Disable r_max_expression_length if it is equal to 0
> > -        rcode += ('' if r_max_expression_length == 0
> > -                  else ',width.cutoff=' + str(r_max_expression_length)) + ')'
> > +        options = {'source="' + escape(filename, '"\\') + '"': True,
> > +                   'blank=' + _map_to_r_bool(r_keep_blank_lines):
> > +                       r_keep_blank_lines is not None,
>
> That would be better
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/coala-analyzer/coala-bears/pull/525/files/72dad32bf787c51b55a67dd0146ecbb675de02a4#r67421605>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJ9xosf01UlHaUBtyE0xhXeEjF9r7uSxks5qMbc-gaJpZM4I14vI>
> .
>